comparison.go

```diff
@@ -82,34 +42,48 @@ func (f *Filter) JSON() string {
 	for _, c := range f.Children {
 		children = append(children, c.JSON())
 	}
+	if f.OrderBy != "" {
+		orderBy := fmt.Sprintf("\"+order_by\": \"%s\"", f.OrderBy)
+		order := fmt.Sprintf("\"+order\": \"%s\"", f.Order)
+		if f.Operator == "" {
+			return fmt.Sprintf("%s, %s, %s",
+				strings.Join(children, ", "), orderBy, order)
+		}
+		return fmt.Sprintf("\"%s\": [%s], %s, %s", f.Operator,
+			strings.Join(children, ", "), orderBy, order)
+	}
+	if f.Operator == "" {
+		return strings.Join(children, ", ")
+	}
 	return fmt.Sprintf("\"%s\": [%s]", f.Operator, strings.Join(children, ", "))
 }
 
-type Comparison struct {
+type Comp struct {
 	Column   string
-	Operator ComparisonOperator
+	Operator string
 	Value    interface{}
 }
 
-func (c *Comparison) GetChildren() []FilterNode {
+func (c *Comp) GetChildren() []FilterNode {
 	return []FilterNode{}
 }
 
-func (c *Comparison) JSON() string {
+func (c *Comp) JSON() string {
```

Comment:
There is an opportunity to use MarshalJSON to get some of this for free and have a reusable interface (rather than exporting a new JSON function that doesn't emit actual JSON).

https://pkg.go.dev/encoding/json